monkey/agent_plugins/build_plugin.sh

```diff
@@ -57,6 +57,10 @@ from pathlib import Path
 
 config_schema = {"type": "object"}
 
+if Path("${PLUGIN_PATH}/config-schema.json").exists():
+  print("\033[0m\033[91m Skipping generating config-schema. Reason: config_schema.json already exists \033[0m")
+  exit
```

Comment:
It seems to me like this would exit the script entirely, and wouldn't generate the plugin at all if the config schema already exists. Maybe instead we should extract the code that generates the schema into a function and call that conditionally?

Reply:
+1. I think this works because it's the last thing this script does, but it could lead to bugs in the future if that changes.